propn-ner-comparator/parse.py

```diff
@@ -0,0 +1,41 @@
+import typer
+from conllu import parse
+from pathlib import Path
+from sklearn.metrics import confusion_matrix
+
+def parse_compare(conllu_file: Path, iob_file: Path):
+    
+    conllu_sentences = parse(conllu_file.read_text())
+    iob_sentences = parse(iob_file.read_text())
+
+    n_tokens = 0
+    n_misaligned = 0
+
+    propn_arr = []
+    tagged_arr = []
+
+    for conllu_s, iob_s in zip(conllu_sentences, iob_sentences):
+        for conllu_tok, iob_tok in zip(conllu_s, iob_s):
+            n_tokens += 1
+            if str(conllu_tok) != str(iob_tok):
+                n_misaligned += 1
+            else:
+                is_propn = (conllu_tok['upos'] == 'PROPN')
+                is_tagged = (iob_tok['lemma'] != 'O')
```

Comment:
@mayhewsw This effectively means that the `OTH` labels are considered as well. That might not make the most sense, so I am again happy to redo this.

Reply:
I'm not opposed to this, but it's good to know. The `OTH` labels are used pretty inconsistently (even in my own annotations, this is the case) so maybe this will cause some problems, but we can leave it for now.